doc/api/assert.markdown

```diff
@@ -2,8 +2,9 @@
 
     Stability: 2 - Stable
```

Comment:
Yeah, I remember that too. Might be good to run this by the TSC in general... I'll tag it tsc-agenda.